examples/zookeeper.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+import logging
+
+import asab
+
+L = logging.getLogger(__name__)
+
+
+class MyApplication(asab.Application):
+	'''
+	Run this with:
+	python3 ./zookeeper.py -c ./zookeeper.conf
+
+	It downloads the configuration file from a zookeeper
+	'''
+
+
+	def __init__(self):
+		super().__init__()
```

Comment:
Print the configuration (use pprint - this is allowed in examples) ... this example is otherwise a bit empty.